monitoring/config-generator/src/generate.js

```diff
@@ -36,6 +37,21 @@ function generate () {
       'kube-namespaces': {
         describe: 'comma separated list of namespaces to monitoring in the `kubernetes` deployment, this is needed for certain special deployments, it defaults to an empty array.',
         type: "string"
+      },
+      'prometheus-storagePath': {
+        describe: 'the path to the TSDB folder',
+        default: '/data/promtheus',
+        type: 'string'
+      },
+      'prometheus-prefix': {
+        describe: 'useful for running prometheus GUI as a subpath , example: /prometheus',
+        default: '/',
+        type: 'string'
+      },
+      'prometheus-externalUrl': {
+        describe: 'external URL for the promtheus service',
+        default: 'http://localhost:9090',
+        type: 'string'
```

Comment:
`kube-longterm` should be documented here, yeah?

Reply:
@iameli done :+1: